cluster/cluster_members.go

```diff
@@ -73,18 +74,19 @@ func (c InternalClusterMember) ToAPI() (*internalTypes.ClusterMember, error) {
 			Address:     address,
 			Certificate: *certificate,
 		},
-		Role:          string(c.Role),
-		SchemaVersion: c.Schema,
-		LastHeartbeat: c.Heartbeat,
-		Status:        internalTypes.MemberUnreachable,
+		Role:                  string(c.Role),
+		SchemaInternalVersion: c.SchemaInternal,
+		SchemaExternalVersion: c.SchemaExternal,
+		LastHeartbeat:         c.Heartbeat,
+		Status:                internalTypes.MemberUnreachable,
 	}, nil
 }
 
 // UpdateClusterMemberSchemaVersion sets the schema version for the cluster member with the given address.
 // This helper is non-generated to work before generated statements are loaded, as we update the schema.
-func UpdateClusterMemberSchemaVersion(tx *sql.Tx, version int, address string) error {
-	stmt := "UPDATE internal_cluster_members SET schema=? WHERE address=?"
-	result, err := tx.Exec(stmt, version, address)
+func UpdateClusterMemberSchemaVersion(tx *sql.Tx, internalVersion int, externalVersion int, address string) error {
```

Comment:
would it be a problem to change these to be uints to better reflect what they are?

Reply:
And if they are coming to/from the db they should probably be uint64.